tests/rip7560/test_rip7560.py

```diff
@@ -37,6 +38,33 @@ def test_eth_sendTransaction7560_valid1(w3, wallet_contract, tx_7560):
     w3.eth.get_transaction(rethash)
 
 
+def test_eth_sendTransaction7560_valid_with_paymaster_postop(w3, tx_7560):
```

Comment:
for completeness: missing test that no context == no postop

Reply:
Sure

Reply:
Added